src/main/scala/App.scala

```diff
@@ -159,5 +162,47 @@ object App extends ZIOAppDefault:
       _ <- printLine(s"1 https zio fixed pool = ${onePool._1.toMillis}ms / req")
       tenPool <- zioReq(urlHttps).repeatN(10).provide(clientLayer).timed
       _ <- printLine(s"10 https zio fixed pool = ${tenPool._1.dividedBy(10).toMillis}ms / req")
+
+      _ <- printLine("\n--------------------------------------------------")
+      _ <- printLine("TOTAL: contains client&ZIO startup/shutdown, similar to cli launch")
+      _ <- printLine("Diff : difference between 1st and 2nd request")
+
+      totalTime <- req2(urlHttps, "ZIO Client.default").provide(Client.default).timed
+      _ <- printLine(s"TOTAL = ${totalTime._1.toMillis}ms")
+
+      totalTime <- req2(urlHttps, "ZIO Client pooled").provide(Client.default).timed
```

Comment:
What is the default for pools in `Client.default`?  We should probably put a bit more on that in the description.

Reply:
I do not analyze details at this point, things change with newer versions, then the fine-tuning comes.